src/renderers/dom/shared/CSSPropertyOperations.js

```diff
@@ -106,6 +119,10 @@ if (__DEV__) {
     } else if (badStyleValueWithSemicolonPattern.test(value)) {
       warnStyleValueWithSemicolon(name, value);
     }
+
+    if (Number.isNaN(value)) {
```

Comment:
Number is not being imported at the top of this file.  I think you'll need to create a Number polyfill module (ie. just copy-paste the one from the MDN docs) and import it in order to support legacy browsers.